### DISCUSSION TITLE
Daniel Roe - Nuxt.js, unjs

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#187] [Announcements]
- https://twitter.com/danielcroe
- https://twitter.com/nuxt_js
- https://nuxt.com/
- https://twitter.com/nuxtstudio
- https://nuxt.studio/
- https://regexp.dev/
- https://twitter.com/unjsio
- https://github.com/unjs
- https://roe.dev/

## Topics

- Nuxt/Vue
- Nuxt studio
- UnJs

### Background

- [x] How did you get into programming and end up doing so much open source?

### Nuxt

- [x] What do you like about vue that makes it better for application development for you?
- [x] What is nuxt and why would I want to use it?
- [x] How does it relate to next.js?
- [x] Next support a lot of different kinds of rendering, does nuxt offer similar?
- [x] Where do you deploy? Is there a provider that suits nuxt best?
- [x] Does it have API routes?
- [x] What are some things you can do in nuxt that aren't available in next?
- [x] Whats your favorite thing that you have contributed to the project?

### Devtools

- [x] What dev tools does nuxt provide? How do they differ from other frameworks?
- [x] How do the devtools help you manage and update your project?
- [x] They provide lots of info (hook usage, modules, etc) Have these tools ever helper you debug a problem?

### Studio

- [x] What is nuxt studio?

### unJS

- [x] What is unJS and what is the group's goal?
- [x] Which packages do you think are cool? Unplugin is super cool!

### Future

- [x] Does nuxt have any cool new features planned?
- [ ] What do you think the future of developing for the web looks like? Does it have more GUI based community tools?

**Comment by just-be-dev** [comment]
Tooltips:
- https://expensive.toys/toys/chromadynamica-manipulable (by Artur Bień)
- https://www.shapr3d.com/

**Comment by hipstersmoothie** [comment]
https://github.com/vantezzen/auto-form
https://github.com/dan-lee/timescape

**Comment by danielroe** [comment]
[sli.dev](http://sli.dev/)
alacritty
plash
raycast
